faf.spec.in

```diff
@@ -459,7 +546,11 @@ A plugin for %{name} implementing checking of repositories
 %package bugtracker-bugzilla
 Summary: %{name}'s bugzilla support
 Requires: %{name} = %{faf_version}
+%if %{with python3}
+Requires: python3-bugzilla >= 2.0
```

Comment:
And here.

Reply:
I keep the version condition here because python3-bugzilla < 2.0 was built in Fedora.